src/app/app-menu/app-menu.component.html

```diff
@@ -23,9 +24,10 @@
 </div>
 <ul [hidden]="!logged" id="app-nav-mobile" class="sidenav">
   <ng-container *ngIf="logged">
-    <li *ngIf="checkVisibility('/qr-code')"><a routerLink="/qr-code" class="sidenav-close main-color">QR</a></li>
-    <li *ngIf="checkVisibility('/assistants')"><a routerLink="/assistants" class="sidenav-close main-color">Assistants</a></li>
-    <li><a routerLink="/account" class="sidenav-close main-color">{{ 'ACCOUNT.TITLE' | translate }}</a></li>
-    <li><a class="sidenav-close main-color" (click)="auth.logout()">{{ 'LOGOUT.TITLE' | translate }}</a></li>
+      <li *ngIf="checkVisibility('/qr-code')"><a routerLink="/qr-code" class="sidenav-close main-color">{{'QR_CODE.TITLE' | translate }}</a></li>
+      <li *ngIf="checkVisibility('/assistants')"><a routerLink="/assistants" class="sidenav-close main-color">{{'ASSISTANTS.TITLE' | translate }}</a></li>
+      <li *ngIf="checkVisibility('/schedule')"><a routerLink="/schedule" class="sidenav-close main-color">{{'SCHEDULE.TITLE' | translate }}</a></li>
+      <li><a routerLink="/account" class="sidenav-close main-color">{{ 'ACCOUNT.TITLE' | translate }}</a></li>
+      <li><a class="sidenav-close main-color" (click)="auth.logout()">{{ 'LOGOUT.TITLE' | translate }}</a></li>
```

Comment:
Is it possible to remove the extra indent and leave all these lines as before? Or Prettier is formatting this like that?